src/components/carousel/README.md

```diff
@@ -11,6 +11,9 @@ controls and indicators. Please be aware that nested carousels are **not** suppo
                 style="text-shadow: 1px 1px 2px #333;"
                 controls
                 indicators
+                focus
+                wrap
```

Comment:
using just `wrap` is the same thing as not using `wrap` at all... this makes for confusing props and docs.